chapters/inheritance.tex

```diff
@@ -551,7 +551,7 @@ \subsection{Modifiers for Array Elements}\doublelabel{modifiers-for-array-elemen
 \end{lstlisting}
 This implies that \lstinline!b[k].c[i].a[j]=j! and
 \begin{lstlisting}[language=modelica]
-  b[k].c[i].d=i and b[k].p=k
+b[k].c[i].d=i and b[k].p=k
```

Comment:
This was wrong before, it should have been two lstinline fragments, with textual "and" between them.

Reply:
Fixed (slightly more ambitiously) in 566eae7415787cb773b87afdc7eb31f9851b60fa.